java/api/src/main/java/org/ray/api/RayCall.java

```diff
@@ -3845,142 +3845,142 @@ public static void call(RayFuncVoid0 f, CallOptions options) {
     Object[] args = new Object[]{obj0, obj1, obj2};
     return Ray.internal().call(pyRemoteFunction, args, options);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3};
     return Ray.internal().call(pyRemoteFunction, args, null);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, CallOptions options) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, CallOptions options) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3};
     return Ray.internal().call(pyRemoteFunction, args, options);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3, obj4};
     return Ray.internal().call(pyRemoteFunction, args, null);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, CallOptions options) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, CallOptions options) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3, obj4};
     return Ray.internal().call(pyRemoteFunction, args, options);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, Object obj5) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, Object obj5) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3, obj4, obj5};
     return Ray.internal().call(pyRemoteFunction, args, null);
   }
 
-  public static RayObject call(PyRemoteFunction pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, Object obj5, CallOptions options) {
+  public static <R> RayObject<R> call(PyRemoteFunction<R> pyRemoteFunction, Object obj0, Object obj1, Object obj2, Object obj3, Object obj4, Object obj5, CallOptions options) {
     Object[] args = new Object[]{obj0, obj1, obj2, obj3, obj4, obj5};
     return Ray.internal().call(pyRemoteFunction, args, options);
   }
 
-  public static RayPyActor createActor(PyActorClass pyActorClass) {
+  public static <R> RayPyActor createActor(PyActorClass pyActorClass) {
```

Comment:
```suggestion
  public static RayPyActor createActor(PyActorClass pyActorClass) {
```

Reply:
I updated RayCallGenerator to generate generic APIs.